install/kubernetes/cilium/charts/hubble-relay/values.yaml

```diff
@@ -1,16 +1,29 @@
-image: hubble-relay
+# Configuration for the Hubble CLI
+image:
+  # repository of the docker image
+  repository: hubble-relay
+  # tag is the container image tag to use
+  tag: latest
+  # pullPolicy is the container image pull policy
+  pullPolicy: Always
```

Comment:
@joestringer will this require a change in the Makefile as well? (This looks like a deja vu)

Reply:
yeah. PR #11837 makes the overall changes necessary, we just need to add this extra chart path to the makefile variable. Let's aim to merge that one first then it's an easy change on top.